rest/pom.xml

```diff
@@ -202,13 +202,12 @@
     </plugins>
   </build>
 
-  <profiles>    
+  <profiles>
      <profile>
         <id>auth</id>
         <activation>
             <property>
                 <name>auth</name>
```

Comment:
Before my changes we relied on this property. It's also used for tests to determine if they should run with auth or not. 

I may try to change it. So far I didn't find a way to set a property in a profile.